docker/postlaunchconfig.sh

```diff
@@ -116,8 +116,8 @@ if [ "${LOG_LEVEL}" == "DEBUG" ]; then
     DEFAULT debug True
     openstack-config --set /etc/keystone/keystone.conf \
     DEFAULT insecure_debug True
-    openstack-config --set /etc/keystone/keystone.conf \
-    wsgi debug_middleware True
+    # openstack-config --set /etc/keystone/keystone.conf \
+    # wsgi debug_middleware True
```

Comment:
Leftover comments?

Reply:
fixed in https://github.com/telefonicaid/fiware-keystone-spassword/pull/237/commits/2c8815cfce7c8997622ed1d6bf1215960200af85